app/lib/forum_support/user_topics.rb

```diff
@@ -21,20 +23,18 @@ def create
       @user_topic = UserTopic.new :user => current_user
 
       # ensuring the topic belongs to the site_account forum
-      @user_topic.topic = @forum.topics.find_by_id(params[:user_topic][:topic_id])
+      @user_topic.topic = @forum.topics.find_by(id: params[:user_topic][:topic_id])
```

Comment:
if `user_topic` is not there, it will raise a `NotMethodError` for `nil`, but it has been there for ages and it isn't your fault.